packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -64,6 +64,25 @@
 	}
 }
 
+.block-editor-block-toolbar__block-parent-selector-wrapper {
+	position: absolute;
+	top: -1px;
+	left: -1px;
+	opacity: 0;
+	transition: all 60ms linear;
```

Comment:
Should we add the reduced motion support?

Reply:
Ah yea. Added it.